integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/dataload/TestLoadDataWithCompression.scala

```diff
@@ -528,8 +571,9 @@ class TestLoadDataWithCompression extends QueryTest with BeforeAndAfterEach with
     var exception = intercept[RuntimeException] {
       loadData()
     }
+    // scalastyle:off lineLength
     assertResult("For not carbondata native supported compressor, the result of method getName() should be the full class name. Expected 'org.apache.carbondata.core.datastore.compression.ZstdCompressor', found 'zstd'")(exception.getMessage)
-
+    // scalastyle:on lineLength
```

Comment:
can enable linelength for complete file to avoid changes in line 419 and 624

Reply:
done